spec/requests/qing_groups_spec.rb

```diff
@@ -1,63 +0,0 @@
-require "rails_helper"
```

Comment:
why was this file deleted? because it's a request spec that you are replacing with a  feature spec?